server/protx/data/api/utils/plotly_figures.py

```diff
@@ -103,8 +136,8 @@ def timeseries_histogram(hist_data):
     pluralize = {'county': 'counties', 'tract': 'census tracts'}
     fmt_units = pluralize[hist_data['fig_aes']['geotype']]
 
-    # number of years to cover: always 2011-2019 (for now)
-    years = [i for i in range(2011, 2020)]
+    # number of years to cover: always 2011-2020 (for now)
+    years = [i for i in range(2011, 2021)]
```

Comment:
Note: because of this line, this code must be deployed with a db file containing 2020 census data. This will be `cooks_20220422.db` or newer.